packages/expo/package.json

```diff
@@ -48,13 +48,13 @@
   "devDependencies": {
     "@clerk/types": "^4.0.0-beta.20",
     "@types/base-64": "^1.0.2",
-    "@types/node": "^18.17.0",
+    "@types/node": "^20.11.24",
     "@types/react": "*",
     "@types/react-dom": "*",
     "eslint-config-custom": "*",
-    "expo-auth-session": "^5.0.2",
-    "expo-web-browser": "^12.3.2",
-    "tslib": "^2.3.1",
+    "expo-auth-session": "^5.4.0",
+    "expo-web-browser": "^12.8.2",
```

Comment:
❓ Shouldn't we move those dependencies to `dependencies` instead of dev since they are being used in our code execution, not only in the build or development step of this package?

Reply:
Yeah you are right, we should also do that with `tslib`

Reply:
Sorry scratch the above, these are included as they are included in `peerDependencies`